spec/fixtures/_nodes/7x.json

```diff
@@ -4,7 +4,7 @@
     "successful" : 3,
     "failed" : 0
   },
-  "cluster_name" : "elasticsearch",
+  "cluster_name" : "opensearch",
```

Comment:
Does this path somehow need to differentiate between Elasticsearch and OpenSearch (i.e. what happens when we get to OpenSearch 7x?

Also, I thought we were removing references to 7x in this PR?

Reply:
Yeah. I removed only references in text and tests. The fill name is used somewhere very deep. Will remove this in next PR.